install_custom_module_map/after/Example.xcodeproj

```diff
@@ -53,7 +53,7 @@ Targets:
         Shell Path: "/bin/sh"
         Shell Script: '"${PODS_ROOT}/Target Support Files/Pods-Example/Pods-Example-frameworks.sh"
 
-'
+          '
```

Comment:
does anyone know where this is coming from?

Reply:
ah its the `Shell Script:` which includes a `\n` in it. I am a bit uncertain why the indentation is different now

Reply:
it's actually correct because we _do_ add `\n` .